src/routes/Login/Login.tsx

```diff
@@ -50,9 +50,14 @@ const Login = () => {
         addUser(res.data.user.id, res.data.user.username);
     };
 
-    const handleSubmit = async (email: string, password: string) => {
+    const handleSubmit = async (
+        username: Username,
+        email: string,
+        password: string
+    ) => {
         if (!email && !password) return;
```

Comment:
There should be also a "&& !username" in this if statement. Check the signup page too! :)